packages/api/src/routes/farmRoute.js

```diff
@@ -61,4 +63,11 @@ router.delete(
   farmController.deleteFarm(),
 );
 
+router.get(
+  '/utc_offset_by_range/:min/:max',
+  checkSchedulerJwt,
+  hasTimeNotificationsAccess,
```

Comment:
I'm not sure we want to use the same JWT token payload for requests to this endpoint, since we are not requesting timed notifications in this case. I think using something like`{"requestFarmsInTimeZone": true}` and making middleware to check for that would make more sense. @smattingly what do you think?